### PR TITLE
Add labs interview checklist. Close #49.

### LINKED ISSUE #49: Create bullet point outline of what to cover in labs interview calls

### PR DESCRIPTION
**Pull request by hellyeah**
Closes #49

### CODE REVIEW

**Comment by MaxWofford** [comment]
@hellyeah Is this a duplicate of #64?

**Comment by TheThirdOne** [comment]
@MaxWofford, no, it's a revision that used @zachlatta's comments. Probably should have been fixed with a force push though.

**Comment by TheThirdOne** [comment]
If this is merged in, #64 should be closed though.

File: meta/labs_program/interview.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+3. Imagine you are someone walking into your club for the first time.What's
+   that experience like? How does someone find out about your club? What are
+   the biggest problems you foresee running into?
+4. If your club goes really well, what will that look like?
+5. Why do you want to be part of the labs program? Can you spend 5 hours
+   outside of your club a week?
+
+Dates we need to know:
+-  Club fair?
+-  First club meeting?
```
Comment (zachlatta) [inline meta/labs_program/interview.md]: Can you add a new line after line 13 per https://github.com/hackedu/meta/blob/master/markdown_style_guide.md#lists?

(sorry for the formatting changes!)

File: meta/labs_program/interview.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Here are some things we want you to think about:
+-  Gender diversity in leadership
+-  Greet people at the door
```
Comment (zachlatta) [inline meta/labs_program/interview.md]: Same thing here. Can you add a new line after line 17?

(sorry!)

**Comment by hellyeah** [comment]
See #77

**Pull request closed without merging**